schemas/2016-07-01-preview/Microsoft.ServerManagement.json

```diff
@@ -0,0 +1,135 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-07-01-preview/Microsoft.ServerManagement.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.ServerManagement",
+  "description": "Microsoft.ServerManagement Resource Types",
+  "resourceDefinitions": {
+    "gateways": {
+      "type": "object",
+      "description" : "A Server management tools gateway",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2016-07-01-preview"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.ServerManagement/gateways"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "upgradeMode": {
```

Comment:
should be put in an "properties" definition. like this:
"properties": {
    "upgradeMode": ...
}